src/ott/tools/soft_sort.py

```diff
@@ -450,6 +451,84 @@ def _quantile(
   return apply_on_axis(_quantile, inputs, axis, q, weight, **kwargs)
 
 
+def multiv_cdf_quantile_maps(
+    inputs: jnp.ndarray,
+    targets: Optional[jnp.ndarray] = None,
+    target_sampler: Optional[Callable[[jax.random.PRNGKey, Tuple[int, int]],
+                                      jnp.ndarray]] = None,
+    key: Optional[jax.random.PRNGKey] = None,
```

Comment:
Should be called rng for consistency. Also would type if for now as `jax.random.PRNGKeyArray` (but not crucial given #446 )